### PR TITLE
Update syntax blocks for JSON, Map, Math

### PR DESCRIPTION
**Pull request by Elchi3**
See https://developer.mozilla.org/en-US/docs/MDN/Structures/Syntax_sections for the new guidelines and #2202

### CODE REVIEW

**Review by wbamberg — COMMENTED** [review]
These look fine, except some have semicolons and some don't. Is there a reason here?

The meta-docs also are inconsistent about this: https://developer.mozilla.org/en-US/docs/MDN/Structures/Syntax_sections.

I think I would prefer no semicolons.

File: files/en-us/web/javascript/reference/global_objects/math/imul/index.html [hunk 1]
```diff
@@ -18,8 +18,7 @@
 
 <h2 id="Syntax">Syntax</h2>
 
-<pre
-  class="brush: js">var <var>product</var> = Math.imul(<var>a</var>, <var>b</var>);</pre>
+<pre class="brush: js">Math.imul(a, b);</pre>
```
Comment (sideshowbarker) [inline files/en-us/web/javascript/reference/global_objects/math/imul/index.html:21]:
```suggestion
<pre class="brush: js">Math.imul(a, b)</pre>
```

File: files/en-us/web/javascript/reference/global_objects/math/fround/index.html [hunk 2]
```diff
@@ -20,8 +20,7 @@
 
 <h2 id="Syntax">Syntax</h2>
 
-<pre
-  class="brush: js">var <var>singleFloat</var> = Math.fround(<var>doubleFloat</var>);</pre>
+<pre class="brush: js">Math.fround(doubleFloat);</pre>
```
Comment (sideshowbarker) [inline files/en-us/web/javascript/reference/global_objects/math/fround/index.html:23]:
```suggestion
<pre class="brush: js">Math.fround(doubleFloat)</pre>
```

File: files/en-us/web/javascript/reference/global_objects/map/clear/index.html [hunk 3]
```diff
@@ -19,8 +19,7 @@
 
 <h2 id="Syntax">Syntax</h2>
 
-<pre class="brush: js"><var>myMap</var>.clear();
-</pre>
+<pre class="brush: js">clear();</pre>
```
Comment (sideshowbarker) [inline files/en-us/web/javascript/reference/global_objects/map/clear/index.html:22]:
```suggestion
<pre class="brush: js">clear()</pre>
```

File: files/en-us/web/javascript/reference/global_objects/map/delete/index.html [hunk 4]
```diff
@@ -19,7 +19,7 @@
 
 <h2 id="Syntax">Syntax</h2>
 
-<pre class="brush: js"><var>myMap</var>.delete(<var>key</var>);</pre>
+<pre class="brush: js">delete(key);</pre>
```
Comment (sideshowbarker) [inline files/en-us/web/javascript/reference/global_objects/map/delete/index.html:22]:
```suggestion
<pre class="brush: js">delete(key)</pre>
```

File: files/en-us/web/javascript/reference/global_objects/map/foreach/index.html [hunk 5]
```diff
@@ -20,13 +20,29 @@
...
+
+// Callback function
+forEach(callbackFn)
+forEach(callbackFn, thisArg)
+
+// Inline callback function
+forEach(function callbackFn() { ... })
+forEach(function callbackFn(value) { ... })
+forEach(function callbackFn(value, key) { ... })
+forEach(function callbackFn(value, key, map){ ... })
```
Comment (sideshowbarker) [inline files/en-us/web/javascript/reference/global_objects/map/foreach/index.html:38]:
```suggestion
forEach(function callbackFn(value, key) { ... })
forEach(function callbackFn(value, key, map) { ... })
```

**Review by sideshowbarker — APPROVED** [review]

**Pull request merged**